### ISSUE TITLE
Новый концепт интерфейса.

### ISSUE THREAD

**Issue by YuBabkin** [YuBabkin/Pepelats#2]
Нужен макет юзер-френдли минималистичного интерфейса.

**Comment by YuBabkin** [comment]
Нафиг новый концепт. Дополнить интерфейс новыми элементами. Собрать элементы окна во области, в фреймы по логической связи. Будет область "системных действией", область коротких интервалов, область длинных интервалов. В след.версии добавится область просто рассчёта координат на произвольный момент времени. 
Пока главное окно не станет монтруозным, концепт интерфейса не менять! )

**Comment by YuBabkin** [comment]
Короче, не в ближайшей версии.

**Comment by vetakim** [comment]
- [ ] панель управления переделать, а то какой-то отстой (кнопки переразложить)
- [ ] при добавлении нового графика рисовать рядом с ним название эфемериды и длину периода (если выбран длинный период)
- [ ] решить, что выводить в качестве имени оси y, если меняем эфемериду и кривые копятся, то название оси непонятно чему соответсвтует.

**Comment by YuBabkin** [comment]
Да, кстати, потом по Х будем выводить не порядковый номер фемерид, а время (дату). Но для этого нужен новый функционал, буду над этим работать.

**Comment by YuBabkin** [comment]
Концепт нарисовали на бумажке, пора работать. )))
